### PR TITLE
Submit the final work (third submission)

### PR DESCRIPTION
**Pull request by utsargo**
### Intro 
This is the final output of the capstone project.

### Description
In this project, I tried to follow the given design guideline. I used custom content for the project. I implemented the following things:

- No CSS framework was used.
- Implemented dynamic html with javascript for the featured performers' section.
- Implemented responsive design for multiple breakpoints.
- Used javascript for mobile menu and 'more' button in the featured performers' section.

**In this submission I made the following changes:**

- Deployed from the feature branch.
- Added the dark bar on top of the navbar.
- Used Cocogoose font.
- Changed font color in the banner.
- Changed Join the conference button style for desktop.
- And some other adjustments.
- 
**Changes made after the second review:**
- Added video link into the pull request description.
- Removed footer from the homepage on mobile.
- Added a home link to the mobile menu.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,191 @@
...
+
+If you like this project give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my fellow micronauts.
```
Comment (Gopxfs) [inline README.md:157]: As requested in the project requirements:
image

Please make sure to add the credits to the original author of the design 😉.

> Note: the credits should be in the `Acknowledgments` section

File: README.md [hunk 2]
```diff
@@ -1 +1,191 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](http://portfolio.utsargo.tech/capstone-1/)
+
```
Comment (Gopxfs) [inline README.md:69]: Awesome job deploying your project! Please make sure to deploy the final version of it, as displayed below, the local and the deployed versions are different in some sections 😉:

Deployed:
image

Local:
image

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @utsargo 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Awesome work! Your capstone looks beautiful.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
Hello @utsargo, please notice that you should have all your changes on the `final-submission` branch. It looks like you merged some changes before. Kindly take into account that the `main` branch should be the final version of your application, and should only receive changes after you secure that everything is working fine 😉 (in Microverse's case, after receiving reviews). This will improve a lot your time fixing bugs and maintaining version control. And also, it makes reviews a lot easier 😄.

**Comment by Gopxfs** [comment]
As requested in the project requirements:
image

Please add the link to your video in this PR 😄.

Your video was very good and you sounded confident, well done! Please consider this point that you forgot:
image

**Comment by Gopxfs** [comment]
https://github.com/portfolio-roy/capstone-1/blob/420e81eb9cf32542633c74cc7b1036c5f34cda67/assets/css/style.css#L124-L136

Please make sure to also include the top dark bar at your header, as displayed on the design 😉:
| Behance design  | Your design |
| ------------- | ------------- |
| image  | image  |

**Comment by Gopxfs** [comment]
https://github.com/portfolio-roy/capstone-1/blob/3ea6fc65899273d399fcecb23f56cd9f6fa4db15/assets/css/style.css#L238-L249
To follow the original design, please apply the COCOGOOSE font to your title 😉.

**Comment by Gopxfs** [comment]
https://github.com/portfolio-roy/capstone-1/blob/420e81eb9cf32542633c74cc7b1036c5f34cda67/assets/css/style.css#L259-L262

To match the Behance design, please apply the color `#272a31` to this paragraph 😉.
image

**Comment by Gopxfs** [comment]
https://github.com/portfolio-roy/capstone-1/blob/3ea6fc65899273d399fcecb23f56cd9f6fa4db15/index.html#L132

Kindly take into account that the orange button should be displayed only on the mobile version. On the desktop, you should have this simple `a` tag:
| Behance design  | Your design |
| ------------- | ------------- |
| image  | image  |

**Comment by Gopxfs** [comment]
https://github.com/portfolio-roy/capstone-1/blob/3ea6fc65899273d399fcecb23f56cd9f6fa4db15/index.html#L13-L15

Amazing job using the `nav` tag in the correct way! Keep in mind that the logo should be outside the `nav`, but still inside your `header` 😉. By doing this, you will improve your page navigation and accessibility.
[Learn more here.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)

**Comment by Gopxfs** [comment]
https://github.com/portfolio-roy/capstone-1/blob/main/about.html#L66-L135

Please make sure to remove all the commented code from your project 😉. This will make your project more clear and easier to maintain.

**Comment by Gopxfs** [comment]
Your first capstone looks outstanding! Kudos to you, keep up the good work 😉. Just improve on these details and you are good to go. Also, be careful with the branch on your next big projects. Never merge an unpolished feature to your final branch, in this case, the `main`. Cheers and keep coding 😄.

File: README.md (hunk 2 above)
Comment (utsargo) [reply README.md:69]: The site is deployed from the main brach. So, once I get passed the review and merge the repo, changes will be there.

**Comment by utsargo** [comment]
> 

I'll keep that in mind.

**Comment by utsargo** [comment]
@Gopxfs COCOGOOSE font is not open source. In the project requirements, there was a recommendation for the Lato font. How can I use a proprietary font without getting it legally?

**Comment by Gopxfs** [comment]
> @Gopxfs COCOGOOSE font is not open source. In the project requirements, there was a recommendation for the Lato font. How can I use a proprietary font without getting it legally?

In case you don't find the font available for free, you can use a similar one, that is ok 👍.

File: index.html [hunk 3]
```diff
@@ -197,5 +197,7 @@ <h2 class="m-0 gray">
     </footer>
```
Comment (jerryowusu) [inline index.html:200]: [OPTIONAL] The mobile version of the main page does not have the footer, it ends at the `more` button. Kindly check and fix it in order to conform with the design

 Your design            |  The figma design 
:-------------------------:|:-------------------------:
Screenshot from 2022-11-24 08-35-20  |  Screenshot from 2022-11-24 08-35-09

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @utsargo 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close


 ## Highlights
:heavy_check_mark: All linters passing
:heavy_check_mark: Project successfully deployed
:heavy_check_mark: Excellent and neat code

 ## Required Changes ♻️

- [x] As requested in the project requirements below, kindly provide a link of your project video documentation in your pull request description. The video should contain a description of the project, a demo of the project features as well as highlights of some interesting piece of code or something you built that you are very proud of. 

video documentaion


_Check the comments under the review._


## Optional suggestions

- [x] [OPTIONAL] There is no link to help you navigate from the about page to the main page on the mobile version currently unless you switch to the deskstop version and click on the logo (which is not available in the mobile version). Kindly provide a link to navigate from the about page to the main page in the mobile version to make your work more proffessional and up to standard. :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Hey @utsargo 👋
# Status: Approved :green_circle: :1st_place_medal: :100:
success

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:


## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github flow
✔️ Following Linters
✔️ Descriptive Readme
✔️ Following requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**